docs/concepts/vega-chain.md

```diff
@@ -4,76 +4,89 @@ title: Vega Chain
+All governance voting and validator nominations happen exclusively on the Vega chain. 
 
-### Nominating validators
-Using tokens to nominate validators keeps the decentralised network functioning. 
+:::info
+Ethereum gas fees are only incurred in the process of associating tokens to a Vega key and transferring rewards from a Vega key to an Ethereum address.
+:::
 
-Tokenholders can nominate validators to encourage a diverse set of reliable nodes running the network, and to give the community the opportunity to disincentivise and/or remove bad validators.
+The Vega protocol listens for stake events from staking bridges. Currently there are two bridges: one for staking unlocked, freely tradeable tokens, and one that connects to the vesting contract for locked, untradeable tokens. 
 
-When a tokenholder chooses a validator (or validators) to nominate with their tokens, the amount is immediately subtracted from their available balance, and is used at the start of the next epoch to actively nominate those validator(s). 
+* Staking **unlocked tokens** uses the staking bridge contract. The staking bridge contains functions enabling users to deposit, remove, and transfer stake by notifying the staking bridge what tokens are associated to which Vega key. 
 
-:::info
-VEGA tokenholders can use [token.vega.xyz](https://token.vega.xyz) to associate their tokens and nominate validators. A Vega Wallet and Ethereum wallet are both required. CoinList custodial users should confirm with CoinList how staking works for them.
+* When staking **locked tokens**, the Vega node interacts with the ERC20 vesting contract, which holds tokens that are locked per a vesting schedule, and provides the same utility as the staking bridge smart contract. This allows locked tokens to be used for staking and governance while not being freely tradeable. 
+
+Whether tokens are unlocked or locked, the bridge events make the Vega network of how many tokens a given party has associated and/or unassociated.
+
+All events (including the above, plus stake per validator and others) are only registered after a certain number of block confirmations, as defined by the network parameter `blockchains.ethereumConfig`. 
+
+:::note Further reading
+**[Staking Bridge contracts](https://github.com/vegaprotocol/Staking_Bridge)** - on Vega's staking bridge GitHub repository.
 :::
 
-### Maximum stake per validator
-Each validator has a maximum amount of stake that they can accept. During restricted mainnet, this will be the same amount for all validators. 
+### Spam protection
+To avoid fragmentation or spam, there is a minimum delegateable stake that defines the smallest unit (fractions of) tokens that can be used for nomination, defined by the network parameter `validators.delegation.minAmount`. 
```

Comment:
minAmount is not a very strong spam protection because the transaction will be processed, it will just be declined. There are better ones employed:
1) you need to have a balance of `spam.protection.delegation.min.tokens` to be able to submit a delegation transaction (I think that's currently 0.1 VEGA on mainnet) 
2) you can't run more than `spam.protection.max.delegations delegation` transactions per day.